tools/cli/go-whisk/whisk/api.go

```diff
@@ -257,3 +338,245 @@ func (s *ApiService) Delete(api *Api, options *ApiOptions) (*http.Response, erro
     return nil, nil
 }
 
+/////////////
+// V2 Cmds //
+/////////////
+func (s *ApiService) ListV2(apiListOptions *ApiListRequestOptions) (*ApiListResponseV2, *http.Response, error) {
+    route := "web/whisk.system/apimgmt/getApi.http"
+
+    routeUrl, err := addRouteOptions(route, apiListOptions)
+    if err != nil {
+        Debug(DbgError, "addRouteOptions(%s, %#v) error: '%s'\n", route, apiListOptions, err)
+        errMsg := wski18n.T("Unable to add route options '{{.options}}'",
+            map[string]interface{}{"options": apiListOptions})
+        whiskErr := MakeWskErrorFromWskError(errors.New(errMsg), err, EXITCODE_ERR_GENERAL, DISPLAY_MSG,
+            NO_DISPLAY_USAGE)
+        return nil, nil, whiskErr
+    }
+    Debug(DbgInfo, "Api GET/list route with api options: %s\n", routeUrl)
+
+    req, err := s.client.NewRequestUrl("GET", routeUrl, nil, DoNotIncludeNamespaceInUrl, AppendOpenWhiskPathPrefix, EncodeBodyAsJson)
+    if err != nil {
+        Debug(DbgError, "http.NewRequestUrl(GET, %s, nil, DoNotIncludeNamespaceInUrl, AppendOpenWhiskPathPrefix, EncodeBodyAsJson) error: '%s'\n", routeUrl, err)
```

Comment:
@mdeuser, I believe the current pattern is very wrong and leads to issues where debug/error messages become invalid. As we have seen here, method names may change and duplicate messages are not updated, breaking the current system. Also, all of the duplicate debug and error messages make the code unnecessary verbose.